1-js/06-advanced-functions/04-var/article.md

```diff
@@ -205,7 +205,7 @@ sayHi();
 
 Because all `var` declarations are processed at the function start, we can reference them at any place. But variables are undefined until the assignments.
 
-In both examples above, `alert` runs without an error, because the variable `phrase` exists. But its value is not yet assigned, so it shows `undefined`.
+In both examples above, `alert` runs without an error, because the variable `phrase` exists, but its value is not yet assigned, so it shows `undefined`.
```

Comment:
I think punctuation should be removed:

>In both examples above, `alert` runs without an error because the variable `phrase` exists but its value is not yet assigned, so it shows `undefined`.